content/contribute/index.adoc

```diff
@@ -0,0 +1,24 @@
+---
+title: "Contribute"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+menu:
+  main:
+    weight: 30
+---
+
+## Reporting issues and feature requests
```

Comment:
asciidoc uses the character "==" instead of "##" ?

Reply:
You can use both, I think that this only impact if you have the toc for the index content but I not tested